packages/docs-app/src/tags/reactExamples.ts

```diff
@@ -36,6 +36,6 @@ function addPackageExamples(
 
 addPackageExamples("core", CoreExamples as any);
 addPackageExamples("datetime", DateExamples as any);
-addPackageExamples("labs", LabsExamples as any);
```

Comment:
No labs examples at all, anymore?

Reply:
Yep no more examples at all, because there's no components, or subitems in labs. It's just an empty shell for now.